source/download/index.html.haml

```diff
@@ -147,7 +147,7 @@ authors: bproffitt, dneary, knesenko, mburns, sandrobonazzola, theron
   |Red Hat Enterprise Linux 4|32-bit, 64-bit||Yes|
   |Red Hat Enterprise Linux 5|32-bit, 64-bit||Yes|
   |Red Hat Enterprise Linux 6|32-bit, 64-bit||Yes|
-  |Red Hat Enterprise Linux 7|32-bit, 64-bit||Yes|
+  |Red Hat Enterprise Linux 7|64-bit||Yes|
```

Comment:
Looks like CentOS 7 i386 (32 bit) still exist:  http://mirror.centos.org/altarch/7/os/

Reply:
Didn't know about it, yet oVirt still does not offer it as an option in the VM options dialog so I'd say it's on the unsupported side of things:
https://github.com/oVirt/ovirt-engine/blob/master/packaging/conf/osinfo-defaults.properties